app/views/editor/rich.html.erb

```diff
@@ -538,7 +538,7 @@
     $('.wk-wysiwyg')
         .atwho(hashtags_config)
         .atwho(emojis_config)
-        //.atwho(at_config) // off for performance reasons for now
+        .atwho(at_config) // off for performance reasons for now
```

Comment:
@BajajSmriti Can you remove the comments? Thanks!

Reply:
```suggestion
        .atwho(at_config)
```